src/event.c

```diff
@@ -123,6 +123,10 @@ void xmpp_run_once(xmpp_ctx_t *ctx, const unsigned long timeout)
 		if (ret < 0 && !tls_is_recoverable(tls_error(conn->tls))) {
 		    /* an error occured */
 		    conn->error = tls_error(conn->tls);
+		    if( conn->error == 5 )
+			xmpp_debug(ctx, "xmpp", "Unrecoverable TLS write error, syscall %s.", strerror(errno));
```

Comment:
Just ignore it. That's the remains of debugging on lossy connection. I use openssl, so, just made this dirty prints.